web/src/components/pages/contribution/contribution.tsx

```diff
@@ -440,92 +375,115 @@ class ContributionPage extends React.Component<Props, State> {
+                      className={
+                        'card card-dimensions ' + (isActive ? '' : 'inactive')
+                      }
+                      style={{
+                        transform: [
+                          `scale(${isActive ? 1 : 0.9})`,
+                          `translateX(${
+                            (document.dir == 'rtl' ? -1 : 1) *
+                            (i - activeSentenceIndex) *
+                            -130
+                          }%)`,
+                        ].join(' '),
+                        opacity: i < activeSentenceIndex ? 0 : 1,
+                      }}>
+                      <div style={{ margin: 'auto', width: '100%' }}>
+                        {sentence?.text}
+                        {sentence?.taxonomy ? (
+                          <div className="sentence-taxonomy">
+                            <Localized id="target-segment-generic-card">
+                              <span className="taxonomy-message" />
                             </Localized>
-                          </StyledLink>
-                        </div>
-                      ) : null}
+                            <StyledLink
+                              className="taxonomy-link"
+                              blank
+                              href={`${URLS.GITHUB_ROOT}/blob/main/docs/taxonomies/${sentence.taxonomy.source}.md`}>
+                              <ExternalLinkIcon />
+                              <Localized id="target-segment-learn-more">
+                                <span />
+                              </Localized>
+                            </StyledLink>
+                          </div>
+                        ) : null}
+                      </div>
                     </div>
-                  </div>
-                );
-              })}
+                  );
+                })}
+              </div>
             </div>
-          </div>
-
-          <div className="pills">
-            <div className="inner">
-              {this.isDone && (
-                <div className="review-instructions">
-                  <Localized id="review-instruction">
-                    <span />
-                  </Localized>
-                </div>
-              )}
-              {pills.map((pill, i) =>
-                pill({
-                  isOpen: this.isDone || selectedPill === i,
-                  key: i,
-                  num: i + 1,
-                  onClick: () => this.selectPill(i),
-                  onShare: this.toggleShareModal,
-                  style:
-                    selectedPill !== null &&
-                    Math.abs(
-                      Math.min(Math.max(selectedPill, 1), pills.length - 2) - i
-                    ) > 1
-                      ? { display: 'none' }
-                      : {},
-                })
-              )}
+          )}
+
+          {shouldShowFirstCTA || shouldShowSecondCTA ? (
```

Comment:
if there a lot of bespoke conditions, it can make the code easier to read to create the conditions as separate variables and then use the variable in the condition

eg
```
const shouldShowCTA = shouldShowFirstCTA || shouldShowSecondCTA;
if(showShouldCTA){}
```

Reply:
This kind of approach can then be applied consistently across all conditions/components